pkg/domain/infra/abi/play_utils.go

```diff
@@ -14,3 +17,12 @@ func getSdNotifyMode(annotations map[string]string, name string) (string, error)
 	}
 	return mode, define.ValidateSdNotifyMode(mode)
 }
+
+// getAutoUpdateAnnotation returns the auto-update policy.  The name is
+// optional.
+func getAutoUpdateAnnotation(annotations map[string]string, name string) (autoupdate.Policy, error) {
+	if len(name) == 0 {
+		return autoupdate.LookupPolicy(annotations[autoupdate.Label])
+	}
+	return autoupdate.LookupPolicy(annotations[autoupdate.Label+"/"+name])
+}
```

Comment:
is `getAutoUpdateAnnotation` only used by tests?  Would it make sense to move it to the `_test.go` file?

Reply:
Nice catch.  I forget to plug it in and it turns out the backend already handles the auto-update label.  I'll drop the commit.